src/UBAFeeCalculator/UBAFeeSpokeCalculator.ts

```diff
@@ -8,6 +8,11 @@ type TokenRunningBalanceWithNetSend = TokenRunningBalance & {
   netRunningBalanceAdjustment: BigNumber;
 };
 
+type FlowFee = {
+  lpComponent: BigNumber;
```

Comment:
nit maybe name this `lpFee` to be more consistent with naming with `balancingFee`

Reply:
Completed here: https://github.com/across-protocol/sdk-v2/pull/217/commits/ff3de28daf7fab5f87b37ea10be4b69748fa89bd